CGATPipelines/pipeline_enrichment.py

```diff
@@ -1,8 +1,143 @@
-"""===========================
-Pipeline Enrichment
-===========================
-
-
+'''
```

Comment:
Please don't copy and paste. The header of `pipeline_enrichment.py` should be kept as it was.